packages/client/src/clients/guide/client.ts

```diff
@@ -160,6 +160,13 @@ const predicate = (
     return true;
   }
 
+  // If we're in debug mode, replicate some server side filtering
+  if (debug.forcedGuideKey) {
+    if (!guide.active) {
+      return false;
+    }
+  }
```

Comment:
```suggestion
  if (!guide.active) {
    return false;
  }
```
I think you can just apply this logic generally regardless of normal vs debug mode.